### PR TITLE
Feature/menu

### PR DESCRIPTION
**Pull request by drews-believer**
added tests.

### CODE REVIEW

File: src/components/App/MenuContainer/MenuContainer.tsx [hunk 1]
```diff
@@ -99,28 +101,28 @@ const MenuContainer = () => {
...
+          <Switch>
+            <BrowserRouter>
+              {initialNestedMenu.map((item: ReactElement) => {
+                return item;
+              })}
+            </BrowserRouter>
+          </Switch>
+        )}
+      </StyledMenuContainer>
+    </BrowserRouter>
```
Comment (letrylesha) [inline src/components/App/MenuContainer/MenuContainer.tsx]: Remove please

File: src/components/App/MenuContainer/NavigationMenuList/NavigationMenuItem/NavigationMenuItem.tsx [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+  path: string;
+  active: boolean;
+  setActive(elem: boolean): void;
+  isNestedItem?: string;
+  addNestedMenu?(elem: NestedMenuType): void;
 }
 
 const MenuItem: React.FC<Props> = (props: Props) => {
-    const Item = styled.li`
+  const Item = styled.li`
```
Comment (letrylesha) [inline src/components/App/MenuContainer/NavigationMenuList/NavigationMenuItem/NavigationMenuItem.tsx:23]: move in another file, ...Styles.ts

**Review by letrylesha — APPROVED** [review]

**Review by danill1278 — COMMENTED** [review]
need to add custom link)

**Review by yanosh-almashi — APPROVED** [review]

**Review by danill1278 — APPROVED** [review]

File: src/components/App/MenuContainer/MenuContainer.tsx [hunk 3]
```diff
@@ -1,15 +1,125 @@
...
+  justify-content: space-between;
+  height: 100vh;
+  text-align: center;
+  width: 90px;
+  background-color: #fefefe;
+  border-right: 2px solid #f8f8f9;
+  color: #9ba6b2;
+  box-shadow: 10px 0px 15px -15px rgba(0, 0, 0, 0.72);
+`;
+
```
Comment (CBond05) [inline src/components/App/MenuContainer/MenuContainer.tsx:62]: maybe move styled components to another file

**Review by CBond05 — APPROVED** [review]

**Pull request closed without merging**